standard/standard_0.php

```diff
@@ -1386,9 +1387,13 @@ function strcspn(string $string, string $characters, int $offset, ?int $length):
  * @param string $string <p>
  * The string being split up into smaller strings (tokens).
  * </p>
- * @param string|null $token [optional] <p>
+ * @param string|null $token <p>
  * The delimiter used when splitting up str.
  * </p>
  * @return string|false A string token.
  */
-function strtok(string $string, ?string $token): string|false {}
+function strtok(
+    string $string,
+    #[PhpStormStubsElementAvailable(from: '5.3', to: '7.0')] $token,
+    #[PhpStormStubsElementAvailable(from: '7.1')] ?string $token = null
```

Comment:
1) `$token` is nullable only starting 8.0?: https://3v4l.org/EBQM8#v800
2) From 5.4 till 7.4 `$string` was called `$str`. Not a big deal I guess?..

Reply:
1. Actually no. https://3v4l.org/ZTWUb `allowsNull` actually shows that since PHP 8 parameter has nullable typehint `?string`
2. Yep. We care about names only for PHP 8 and later